src/bentoml/_internal/cloud/config.py

```diff
@@ -79,7 +79,7 @@ def get_context(self, context: t.Optional[str] = None) -> CloudClientContext:
             if ctx.name == context:
                 return ctx
         raise CloudRESTApiClientError(
-            f"BentoCloud API token for {context} cloud context is required.",
+            f"No cloud context {context} found",
```

Comment:
Is it because user has not logged in? If so, remind them to login and show a sample command.

Reply:
Yes. may not have this context, we will show sth like that
![image](https://github.com/user-attachments/assets/fb2b9f74-a636-436a-b5c0-7e48cdad4315)